### PR TITLE
Point references to "VFS Team Roster" to atlas

### PR DESCRIPTION
**Pull request by flooose**
Updates all references to VFS Team Roster to point to atlas

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-vfs-team.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ In addition to providing information to Platform and completing necessary tasks
 
 ## Platform Checklist
 The items in this checklist are for Platform to complete. If you are ready to close the ticket and the Platform Checklist section has not been completed, please reach out to Platform via the guidance provided in [Getting help from Platform in Slack](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html).
-- [ ] New VFS Team has been added to the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=0)
+- [ ] New VFS Team has been added to the [VFS Team Roster](https://www.va.gov/atlas/product_directory/team_members)
```
Comment (RachalCassity) [inline .github/ISSUE_TEMPLATE/new-vfs-team.md:84]: New Team should be added to https://www.va.gov/atlas/product_directory/teams, right?

Comment (flooose) [reply .github/ISSUE_TEMPLATE/new-vfs-team.md]: Thank you!

File: .github/ISSUE_TEMPLATE/aws-access-request.yml [hunk 2]
```diff
@@ -132,7 +132,7 @@ body:
       label: "User must exist in a roster before AWS access can be granted. See [AWS access documentation](https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools#AWS-Access-Request) for the steps involved."
       description: Don't fill this part out, please
       options:
-      - label: "Search for user on the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)"
+      - label: "Search for user on the [VFS Team Roster](https://www.va.gov/atlas/product_directory/team_members)"
       - label: "Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)"
       - label: "If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)"
```
Comment (rmtolmach) [inline .github/ISSUE_TEMPLATE/aws-access-request.yml:137]: Change "If user is on a VFS team but not in the VFS Team Roster" as well

Comment (rmtolmach) [reply .github/ISSUE_TEMPLATE/aws-access-request.yml:137]: oh wait 🤔 I see you are keeping the name "VFS Roster" or "VFS Team Roster" and linking to Atlas here and everywhere else in the PR. Is that what we want or should we go all in and just call it "Atlas"?

**Comment by rmtolmach** [comment]
👀 You might want to check with @bdech. I remember him doing this ticket back in October (and then reverting b/c it was too early!) and then I just saw they might be picking it up again during this sprint! https://github.com/department-of-veterans-affairs/va.gov-team/issues/66361#issuecomment-1866221166

File: .github/ISSUE_TEMPLATE/aws-access-request.yml (hunk 2 above)
Comment (RachalCassity) [reply .github/ISSUE_TEMPLATE/aws-access-request.yml]: Search for user on Platform Atlas

**Review by RachalCassity — APPROVED** [review]

**Comment by RachalCassity** [comment]
LGTM

**Comment by flooose** [comment]
> 👀 You might want to check with @bdech. I remember him doing this ticket back in October (and then reverting b/c it was too early!) and then I just saw they might be picking it up again during this sprint! [#66361 (comment)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66361#issuecomment-1866221166)

I've commented their issue and will coordinate with them.

**Comment by rmtolmach** [comment]
> > 👀 You might want to check with @bdech. I remember him doing this ticket back in October (and then reverting b/c it was too early!) and then I just saw they might be picking it up again during this sprint! [#66361 (comment)](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66361#issuecomment-1866221166)
> 
> I've commented their issue and will coordinate with them.

My suggestion: Get Brandon's review and ✅ and call it good!

**Comment by bdech** [comment]
Looks good to me!

**Pull request merged**